stream-chat-android-client/src/main/java/io/getstream/chat/android/client/uploader/FileUploader.kt

```diff
@@ -42,7 +43,7 @@ public interface FileUploader {
         connectionId: String,
         file: File,
         callback: ProgressCallback,
-    ): Result<String>
+    ): Result<UploadedFile>
```

Comment:
Kdocs needs to be updated.
The `Result` doesn't contain the URL of the uploaded file anymore, but a `UploadedFile` instance

Reply:
Updated to reflect the change.